Alloy.Api/Services/AlloyBackgroundService.cs

```diff
@@ -32,20 +32,23 @@ public class AlloyBackgroundService : IAlloyBackgroundService
         private readonly IServiceScopeFactory _scopeFactory;
         private readonly IAlloyEventQueue _eventQueue;
         private readonly IHttpClientFactory _httpClientFactory;
+        private readonly StartupHealthCheck _startupHealthCheck;
```

Comment:
This healthcheck just sets StartupTaskCompleted to true when the bootstrap function completes, and is done.  This is the readiness check